### PR TITLE
feat: Add Home page

### PR DESCRIPTION
**Pull request by jalajcodes**
https://deploy-preview-2--max-quiz.netlify.app/

### CODE REVIEW

File: styles/index.scss [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+  margin: 2rem auto;
+}
+
+.header {
+  max-height: 100vh;
+
+  .cta {
+    display: flex;
+    margin-top: 5rem;
+    // align-items: center;
```
Comment (foolhardy21) [inline styles/index.scss]: hey, you can remove commented code. At line 91 and 136 too.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+          <img src="./images/quiz.svg" alt="" />
+        </div>
+      </div>
+    </header>
+
+    <section class="categories container" id="categories">
+      <h2>Categories</h2>
+        <div class="grid col-autofit">
+            <a href="#" class="category">
+              <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTbYNOlsuCweydQnsZpXH3ovhqaaD1JRDu19A&usqp=CAU" alt="" />
```
Comment (foolhardy21) [inline index.html]: add some text in alt attribute.

**Review by foolhardy21 — APPROVED** [review]
apart from mentioned things. Looks good to merge.

File: styles/index.scss [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+      margin: 0 0.4rem;
+    }
+    a {
+      margin-left: 0.3rem;
+    }
+  }
+  &__links i {
+    font-size: 1.5rem;
+    margin: 0 0.8rem;
+    color: #b8b8b8;
```
Comment (hsnice16) [inline styles/index.scss]: Is this color used at only place? as you are defining color variables for other colors. So to keep consistency, you can declare a variable for this color too.

File: styles/index.scss [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+h2 {
+  margin: 2rem 0;
+}
+
+.container {
+  max-width: 1280px;
+  margin: 2rem auto;
+}
+
```
Comment (hsnice16) [inline styles/index.scss:24]: Nice, you have maintained the spacing between declarations.

**Review by hsnice16 — APPROVED** [review]
Great work. Loved the UI. nice color choice and you have used it more nicely. Just mentioned some suggestions.

**Pull request merged**